SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -332,6 +332,18 @@ ASpatialFunctionalTestFlowController* ASpatialFunctionalTest::GetLocalFlowContro
 	return LocalFlowController;
 }
 
+ESpatialFunctionalTestWorkerType ASpatialFunctionalTest::GetLocalWorkerType()
+{
+	auto* AuxFlowController = GetLocalFlowController();
```

Comment:
Prefer actual type outside of iterations, but not a big deal.

Reply:
We have such huge names that since it's self explanatory I just wanted to make it shorter, but updating with the class.